docs/xdr/features/collect/integrations/endpoint/sekoiaio.md

```diff
@@ -117,40 +123,92 @@ The Endpoint Detection Agent is easy to install on Windows or Linux systems once
     sudo systemctl status SEKOIAEndpointAgent.service
     ```
 
-Once installed, the agent collects, normalizes, and sends event logs to Sekoia.io. The protocol used to send events is HTTPS (443).
+    #### journald configuration
 
-#### journald configuration
+    To get events, the agent pushes rules to the audit framework. By default, **journald**  might listen to the audit socket for events.
 
-To get events, the agent pushes rules to the audit framework. By default, **journald**  might listen to the audit socket for events.
+    To disable audit logging, do the following as root:
 
-To disable audit logging, do the following as root:
-	
-       
-	# Stop listening to audit events
-	systemctl stop systemd-journald-audit.socket 
-	
-	# Disable it to avoid future start
-	systemctl disable systemd-journald-audit.socket
-	
-	# Masking will prevent starting by other services
-	systemctl mask systemd-journald-audit.socket
-	
-	# Restart journald
-	systemctl restart systemd-journald
-       
-       
-A reboot may be necessary if the audit events are still appearing in the logs.
+
+        # Stop listening to audit events
+        systemctl stop systemd-journald-audit.socket 
+
+        # Disable it to avoid future start
+        systemctl disable systemd-journald-audit.socket
+
+        # Masking will prevent starting by other services
+        systemctl mask systemd-journald-audit.socket
+
+        # Restart journald
+        systemctl restart systemd-journald
+
+
+    A reboot may be necessary if the audit events are still appearing in the logs.
+
+=== "MacOs"
+
+    Unzipping SekoiaEndpointAgent.zip Archive
+
+    1.  Open the Archive: Double-click the file. macOS will automatically open it using the Archive Utility.
+
+    2.  Extracting Files: Once the Archive Utility opens, the files will be extracted to the same location as the original archive. You can also specify a different location if needed.
+
+    Move the application from its download folder to the `/Applications` directory.
+
+    Grant full disk access to SekoiaEndpointAgent by following these steps:
+
+    1.  Click on the Apple icon on the top left corner of your screen.
+
+    2.  Select System Preferences.
+
+    3.  Go to Security & Privacy Preferences > Privacy and click Full Disk Access from the left panel.
+
+    4.  Tick the checkbox for SekoiaEndpointAgent. If the lock at the bottom left is locked, click it to unlock the Privacy pane.
+
+        If you can't find SekoiaEndpointAgent in step 4, do the following:
+
+        1.  Click the plus (+) icon and navigate to Applications.
+
+        2.  In the left panel, find SekoiaEndpointAgent and click Open. Make sure SekoiaEndpointAgent is ticked and then close the window.
+
+    Now that `Full Disk Access` is enable, you can install the agent:
+
+    ```shell
+    sudo /Applications/SekoiaEndpointAgent.app/Contents/MacOs/SekoiaEndpointAgent install --intake-key <INTAKE_KEY>
+    ```
```

Comment:
<img width="1087" alt="Capture d’écran 2024-02-02 à 11 21 26" src="https://github.com/SEKOIA-IO/documentation/assets/154240656/42dbc1db-9406-40a6-bf68-28f5fb2a21d6">

Reply:
~ » sudo /Applications/SekoiaEndpointAgent.app/Contents/MacOs/SekoiaEndpointAgent install --intake-key QNrXazxmoqcGf1fhopGKjiVZ4sZDetGg                                                          juliencuny@Laptop255
Password:
sudo: unable to execute /Applications/SekoiaEndpointAgent.app/Contents/MacOs/SekoiaEndpointAgent: Bad CPU type in executable
--------

Reply:
<img width="919" alt="Capture d’écran 2024-02-02 à 11 19 38" src="https://github.com/SEKOIA-IO/documentation/assets/154240656/2f01693d-e23d-43d5-9c84-0f71c03b9578">